luigi/mock.py

```diff
@@ -135,6 +135,10 @@ def rename(self, *args, **kwargs):
     def path(self):
         return self._fn
 
+    @property
+    def fn(self):
```

Comment:
Looks like I can, yep! Had just been following the convention that was already in the module.

Switched over to using `.path` - that works fine. Also removed extra code that had been added in `mock.py`.